frontend/src/components/SideBar.tsx

```diff
@@ -8,169 +8,88 @@
 // or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
 // OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 // limitations under the License.
+import {SideNavigation, SideNavigationProps} from '@awsui/components-react'
 import * as React from 'react'
-import {Link, useLocation} from 'react-router-dom'
-import {setState, useState, isGuest, isUser, isAdmin} from '../store'
-
-// Icons
-import GridOnIcon from '@mui/icons-material/GridOn'
-import ImageIcon from '@mui/icons-material/Image'
-import LayersIcon from '@mui/icons-material/Layers'
-import HomeIcon from '@mui/icons-material/Home'
-import GroupIcon from '@mui/icons-material/Group'
 import {useTranslation} from 'react-i18next'
-import Image from 'next/image'
```

Comment:
todo: since it's the last reference to `next/image` the experimental part of the next.config for images can be safely dropped (in this PR or another one)

Reply:
Will open another PR for this

Reply:
https://github.com/aws-samples/pcluster-manager/pull/337